src/main/java/com/okta/tools/helpers/ProfileHelper.java

```diff
@@ -6,11 +6,15 @@
 import org.apache.commons.lang.StringUtils;
 
 import java.io.IOException;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
 
 public class ProfileHelper {
 
     private final CredentialsHelper credentialsHelper;
     private OktaAwsCliEnvironment environment;
+    private final Pattern assumedRoleUserPattern = Pattern.compile(
+            "^arn:aws:sts::(?<account>\\d{12}):assumed-role/(?<roleName>[^/]*)/(?<userName>.*$)");
```

Comment:
🌱 "parser" for the assumed role ARN. Allows us to get the account number, role name, and user name from the ARN to build a cleaner profile name.